src/pages/CocosGame.tsx

```diff
@@ -60,9 +60,11 @@ const CocosGame: React.FC = () => {
 
   const push = () => {
     const urlParams = new URLSearchParams(window.location.search);
-    history.replace(state.from ?? PAGES.HOME);
     if (!!urlParams.get("isReload")) {
+      history.replace(state.from + "&isReload=true" ?? PAGES.HOME);
```

Comment:
This will fail if `state.from` is null or `state.from` doesn't have '?'(query parameters)